homeassistant/components/switch/smartthings.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
Move this file to `components/smartthings/switch.py`

Reply:
Is this a new pattern for components that have associated platforms?  When would it go under components/switch?  In looking at other components, i.e. Hue, I see their platforms under the type of component.  For example, the hue light is under components/lights not components/hue.  Thanks for the clarification!

Reply:
@andrewsayre See https://github.com/home-assistant/home-assistant/pull/19948

Reply:
@arsaboo Got it!  Does the documentation follow-suite? `switch.smartthings.markdown` to `smartthings.switch.markdown`?

Reply:
Yes documentation follows, although how much Switch documentation is there? Isn't it all just "Smart Things" ?

Reply:
Also, if you rebase you will see Hue under `hue/` ;-)

Reply:
@balloob I see that now! :) The switch docu is just a stub saying to setup the component.

Reply:
Addressed in latest push!  Thanks!